### ISSUE TITLE
Why in gencode.comprehensive.splice.junctions.txt junction start is in the intron and junction end is 2bp in exon?

### ISSUE THREAD

**Issue by naumenko-sa** [naumenko-sa/crt#1]
Look at any junction, i.e.
15	42698142	42700410	CAPN3	ENST00000569136.1	protein_coding
15:42698142 is the first base of the intron
15:42700410 is the second base of the exon

How does it influence annotations, given that we use flank=1

**Comment by ekofman** [comment]
Yes, I noticed this as well. This is making it such that some splice junctions are appearing as "novel" though there are in fact off-by-one-or-two discrepancies. @naumenko-sa Did you try making your own version of gencode.comprehensive.splice.junctions.txt to address this issue?

**Comment by naumenko-sa** [comment]
Hi @ekofman . We have the same issue. Some junctions appear as novel (not annotated by gencode). I think the easiest way to remove some false positives is to use flank=2 in AddJunctionsToDatabase1.py

**Comment by ekofman** [comment]
Yes @naumenko-sa I was able to fix the problem by just being careful about the indexing (for CIGAR strings start index is inclusive but end index is not inclusive), and ensuring that I was simply subtracting two from the junction end.
